docs/msbuild/msbuild-inline-tasks.md

```diff
@@ -18,6 +18,8 @@ MSBuild tasks are typically created by compiling a class that implements the <xr
   
  Starting in .NET Framework version 4, you can create tasks inline in the project file. You do not have to create a separate assembly to host the task. This makes it easier to keep track of source code and easier to deploy the task. The source code is integrated into the script.  
   
+
+ In MSBuild 15.8, the [RoslnCodeTaskFactory](../msbuild/msbuild-roslyncodetaskfactory.md) was added which can create .NET Standard cross-platform inline tasks.  If you need to use inline tasks on .NET Core, you must use the RoslynCodeTaskFactory.
```

Comment:
Is it worth saying this is now the preferred way of doing inline tasks? Is it the new preferred way? :)

Reply:
Its required for inline tasks on .NET Core but existing builds can just stick with `CodeTaskFactory`.